packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import {
+  string,
+  element,
+  object,
+  oneOf,
+  oneOfType,
+  shape,
+  arrayOf,
+} from 'prop-types';
+import styled from '@emotion/styled';
+import { C_LUNAR, C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_HLF, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const [TAG_TEXT_PAD_Y, TAG_TEXT_PAD_X, ROW_SPACING] = [
+  '0.375rem',
+  '0.4375rem',
+  '0.0625rem',
+];
+
+const topicTagParent = `
+  word-break: break-word;
+  margin-top: ${ROW_SPACING};
+  margin-left: ${GEL_SPACING_HLF};
+  margin-right: ${GEL_SPACING_HLF};
+  a {
+    display: flex;
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING};
+    text-decoration: none;
+    color: ${C_EBON};
+
+    &:hover, &:focus {
+      text-decoration: underline;
+    }
+    &:visited {
+      color: ${C_METAL};
+    }
+  }
+
+  a > span {
+    background-color: ${C_LUNAR};
+    padding: ${TAG_TEXT_PAD_Y} ${TAG_TEXT_PAD_X};
+  }
+`;
+
+const topicTagsContainer = `
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: -${ROW_SPACING};
```

Comment:
Not a negative, just to understand, why is negative margin needed here, is it overriding another margin applied elsewhere?

Reply:
In the UX, there is a `1px` gap between each `44px` hit area, and so each topic tag has a `margin-top` of `1px` to give this effect, but then the first row of topic tags will be unnecessarily offset from the top of the container by `1px`, and this negative margin is to compensate 👍 I'm aware it's not a pretty solution, so open to suggestions.

Reply:
Thanks for explaining, all I can think of is something like this: https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child but you don't know how many children are in the first row. You could remove the 1px gap for the children on the first row but I don't know how you could identify them really. This seems fine to me unless there's a css expert about 💡

Reply:
Yeah I was using `:not(:first-of-type)` to remove the margin of the first `TopicTag` until recently, when I found exactly what you described where there are more than 1 `TopicTag`s in the first row, so this is the best I could come up with 👍